### DISCUSSION TITLE
Áru visszavét

### DISCUSSION THREAD

**Discussion by MicraMetripond** [nav-gov-hu/eRECEIPT#24] [Kérdések és válaszok]
Ha egy 5 tételes nyugtáról visszaveszik pl. a 3. tételt (papírnyugta nélkül) akkor hogyan fogják ellenőrizni, hogy ugyanezt a fajta terméket még egyszer ne lehessen visszavenni ugyanezen a nyugtán? Jelenleg a papír alapú nyugtán kézzel rögzítik, hogy a visszavétel már megtörtént, de elektronikus nyugta esetében az hogyan történik majd?

**Comment by renced42** [comment]
Kedves @MicraMetripond 

Jellemzően a visszavételt is bizonylatolás kíséri. Ezekhez a bizonylatolási műveletekhez lehet kapcsolni az alap bizonylat sorszámát amelyre vissza lehet kérdezni a NAV által biztosított interfészen majd. Tehát tudja ellenőrizni az eladó akár az előzményeket is ha szükséges.
